server/src/bots/telegram/utils/messageRegistry.ts

```diff
@@ -1,32 +1,36 @@
 import {getChatId} from "./chat";
+import {logger} from "../../../utils/logger";
 
-export class MessageRegistry
-{
+export class MessageRegistry {
     //TODO: change type to unknown and Handle casting to BotType 
     _bot: any;
-    _CallBackQueryHandlers: { [id: string] : CallBackQueryHandler; } = {};
+    _messageHandlers: { [regexp: string]: CallBackQueryHandler } = {};
 
     constructor(bot) {
         this._bot = bot;
+        this.registerCallBackQuery();
     };
 
-    Register(message: string, callback: (bot: any, message: any, chatId: unknown) => any) : MessageRegistry {
-        this._bot.onText(new RegExp(message, 'g'), (message) => callback(this._bot, message, getChatId(message)));
+    public registerMessageHandler(regexp: string, callback: (bot: unknown, message: unknown, chatId: number) => unknown): MessageRegistry {
+        this._messageHandlers[regexp] = callback;
```

Comment:
As far as I know. When we use this method for subscribing on "onText". we say that on specific type of message which is set by regex under the line we are going to use concrete callback.

A bit different pattern works for when we subscribe for "callback_query". Here we do it only once. And have to handle choosing the callback by ourselves, since there is no parameter like regex for this kind of subscription.

So everything written above is for making sure we both are understanding node telegram API the same way and correctly. 
And my actual proposition is just to remove the line 15.  🇺🇬

Reply:
I see now. You just moved merged it to be a single method.
But my personal opinion is that If api devs made those methods different. It make sense handling them differently and don't mess everything up.
I mean having those handlers registered explicitly in a different way, we will have much clear vision on what user request we actually handle just from code. 
It prolly just my thoughts....

Reply:
reverted back 🥇